src/api.ts

```diff
@@ -48,3 +49,16 @@ export const getChapterVersesResponse = async (
 ) => {
   return fetcher(makeVersesUrl(id, params));
 };
+
+/**
+ * Get the current available translations with the name translated in the current language.
+ *
+ * @param {string} language we use this to get translated names of authors in specific the current language.
+ *
+ * @returns {Promise<TranslationsResponse>}
+ */
+export const getAvailableTranslations = async (language: string): Promise<TranslationsResponse> => {
```

Comment:
`getAvailableTranslations` is bit confusing.  Why not just `getTranslations`?

Reply:
Maybe I can rename it to `getTranslationsResources` since it will match the API path `/resources/translations` and also we can include "resource" terminology as a distinguisher between a translation and a translation of a verse. In my commit, I already introduced a type called `TranslationResource` that represents one item inside the array of translations. What do you think @naveed-ahmad @tifa2UP ?

Reply:
Chatted with @osamasayed, availableTranslations seems to be the clearest option. Let's go with it for now.